### PR TITLE
Use bash script in CI

### PR DESCRIPTION
**Pull request by thvdveld**
Use a bash script in CI. The advantage of this is that we can run this script locally as well, and thus check if a PR would pass the checks or not locally.

### CODE REVIEW

File: ci.sh [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```
Comment (crawford) [inline ci.sh]: Not all distributions install bash to this location.

```suggestion
#!/usr/bin/env bash
```

Comment (thvdveld) [reply ci.sh]: Changed it. Thanks!

**Review by Dirbaio — APPROVED** [review]
Thank you!

bors r+

**Comment by thvdveld** [comment]
bors retry

**Comment by thvdveld** [comment]
I didn't know this was important, but I think CI still needs for bors:
```diff
diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index c87d092..eaa2c3d 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -6,6 +6,13 @@ on:
 name: Test
 
 jobs:
+  tests:
+    runs-on: ubuntu-22.04
+    needs: [check-msrv, test-msrv, test-stable, clippy]
+    steps:
+      - name: Done
+        run: exit 0
+
   check-msrv:
     runs-on: ubuntu-22.04
     steps:
```

**Pull request merged**